package.json

```diff
@@ -22,14 +22,15 @@
 	},
 	"devDependencies": {
 		"@changesets/cli": "^2.27.1",
-		"@sveltejs/eslint-config": "^6.0.4",
+		"@sveltejs/eslint-config": "jrmajor/svelte-eslint-config#eslint-9",
 		"@svitejs/changesets-changelog-github-compact": "^1.1.0",
-		"@typescript-eslint/eslint-plugin": "^7.0.0",
-		"eslint": "^8.56.0",
+		"eslint": "^9.0.0",
 		"eslint-config-prettier": "^9.1.0",
-		"eslint-plugin-svelte": "^2.35.1",
+		"eslint-plugin-svelte": "^2.36.0",
 		"eslint-plugin-unicorn": "^52.0.0",
-		"playwright": "^1.41.0"
+		"globals": "^15.0.0",
```

Comment:
I feel it's a bit surprising to have to install this. Can we make it a regular dependency rather than peer dependency of `@sveltejs/eslint-config`?

Reply:
sveltejs/eslint-config#35